api/v1beta1/kafkacluster_types.go

```diff
@@ -476,6 +476,9 @@ type CommonListenerSpec struct {
 	// under the keystore.jks, truststore.jks, password data fields.
 	// If this field is omitted koperator will auto-create a self-signed server certificate using the configuration provided in 'sslSecrets' field.
 	ServerSSLCertSecret *corev1.LocalObjectReference `json:"serverSSLCertSecret,omitempty"`
+	// SSLClientAuth specifies whether client authentication is required, requested, or not required
```

Comment:
Also mention that if ommited than defaults to `required` (to keep backward compatibility with previous koperator versions as in case of Kafka if omitted than defaults to none)